.travis.yml

```diff
@@ -13,12 +11,11 @@ cache: pip
 install:
   - bash .travis/install_terraform.sh
   - make docker.build
-  - make local.up
 
 script:
-  - docker exec -t woeip.app make quality
-  - docker exec -t woeip.app make static
-  - docker exec -t woeip.app make test
+  - docker-compose -f docker-compose.yml -f docker-compose.travis.yml run app "make quality"
```

Comment:
Interesting way to solve the problem: create a compose configuration specifically for travis. Noting the difference between production and travis compose will help teach me a little more about both- and exactly how gunicorn does its thing.